coffee/slide.coffee

```diff
@@ -13,7 +13,8 @@ document.addEventListener 'DOMContentLoaded', ->
       for target in $(@).attr('data-marp-path-resolver').split(/\s+/)
         $(@).attr(target, resolvePathFromMarp($(@).attr(target)))
 
-    Markdown = new clsMarkdown({ afterRender: clsMarkdown.generateAfterRender($) })
+    twemojiBasePath = Path.resolve(__dirname, '../images/twemoji/') + '/'
```

Comment:
Oh, it requires overriding [twemoji's callback](https://github.com/twitter/twemoji/blob/gh-pages/README.md#callback) to return correctly path in Windows.